_shared_content/develop/guides/formats/best_practices/authentications.md

```diff
@@ -0,0 +1,520 @@
+# Authentications
+
+In order to simplify the recognition and semantic interpretation of authentications, it is necessary to respect some standardization.
+
+An authentication is fully recognized as valid when all of these four fields are completed:
+
+- event.category : with the value `authentication`
+- event.type : with the value `start`
+- action.outcome : with the value `success` if the authentication successed or `failure` else
+
+Other fields are necessary in order to be able to fully describe the authentication. These fields are described in the table below
+
+| Field                       | Description                                                          |
+| --------------------------- | -------------------------------------------------------------------- |
+| event.action                | Logon action type                                                    |
+| event.reason                | Reason type                                                          |
+| sekoiaio.client.name        | Hostname from which authentication is requested                      |
+| sekoiaio.client.domain      | Domain of the host from which authentication is requested            |
+| sekoiaio.client.os.type     | OS type of the host from which authentication is requested           |
+| client.ip                   | IP of the host from which authentication is requested                |
+| client.geo.country_iso_code | ISO COUNTRY CODE of the host from which authentication is requested  |
+| sekoiaio.server.name        | Host name  targeted by the authentication                            |
+| sekoiaio.server.domain      | Host domain targeted by the authentication                           |
+| sekoiaio.server.os.type     | OS type of the host targeted by the authentication                   |
+| server.ip                   | Host ip targeted by the authentication                               |
+| server.geo.country_iso_code | Host geoloc targeted by the authentication                           |
+| process.name                | process name has perfomed authentication (i.e., sshd, kerberos, ...) |
```

Comment:
Becareful our current integrations for Windows seems to put it in `process.name` the `action.properties.LogonProcessName` for events 4624 and 4625. We recommand to keep the original `process.name` from the event instead.